bricksrc/equipment.py

```diff
@@ -514,6 +515,7 @@
                 SKOS.definition: Literal(
                     "An optical instrument to capture still images or record moving images, which are stored on a physical or digital medium."
                 ),
+                "parents": [BRICK.Camera_Equipment]
```

Comment:
Change this to `BRICK.Camera` based on the other comment

Reply:
Done